example/lib/main.dart

```diff
@@ -12,6 +15,11 @@ void main() async {
     await Flame.device.setLandscape();
     await Flame.device.fullScreen();
   }
+
+  BonfireInjector().put((i) => KnightController());
+  BonfireInjector().putFactory((i) => GoblinController());
```

Comment:
I think it would be nice to document the difference between using `putFactory` and `put`

Reply:
I did add inside the these methods a comments explaining more your use. You can take a look?